src/main/java/teammates/logic/api/TaskQueuer.java

```diff
@@ -294,6 +294,21 @@ public void scheduleEmailsForSending(List<EmailWrapper> emails) {
         }
     }
     
+    /**
+     * Schedules production of searchable documents
```

Comment:
This comment does not mention the scope of this operation. Does it produce searchableDocs for all entities in the course? 

Why is it taking only one instructor email? A course can have more than one instructor.